app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -50,6 +50,7 @@
 import android.webkit.WebBackForwardList;
 import android.webkit.WebChromeClient;
 import android.webkit.WebHistoryItem;
+import android.webkit.WebIconDatabase;
```

Comment:
???

Reply:
I've removed unneeded import statement and force pushed